### PR TITLE
[DO-1895] Capitalize country names in DB

### PR DESCRIPTION
**Pull request by Iva-khar**
A Part of Task DO-1895 - Rewrite existing fields stored in lowercase

### CODE REVIEW

File: data_ocean/converter.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ def print_running_times(self):
             self.timer.print_result()
 
     def save_or_get_country(self, name):
-        name = name.lower().strip()
+        name = name.strip()
         if name not in self.all_countries_dict:
```
Comment (roman-tiukh) [inline data_ocean/converter.py:45]: this comparing should be case insesitive

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: location_register/management/commands/capitalize_country_names.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+class Command(BaseCommand):
+    help = 'Capitalize name and name_uk fields in Country'
+
+    def handle(self, *args, **options):
+        self.capitalize_country_names()
+
+    @staticmethod
+    def capitalize_country_names():
+        for name in CAPITALIZE_COUNTRIES_EN:
+            country = Country.objects.filter(name=name).first()
```
[the remark this answers is not in the document]
Comment (Iva-khar) [reply location_register/management/commands/capitalize_country_names.py:15]: done

File: location_register/management/commands/constants.py [hunk 3]
```diff
@@ -0,0 +1,589 @@
...
+	'ukraine': 'Ukraine',
+	'austria': 'Austria',
+	'azerbaijan': 'Azerbaijan',
+	'albania': 'Albania',
+	'anguilla': 'Anguilla',
+	'argentina': 'Argentina',
+	'bahamas': 'Bahamas',
+	'barbados': 'Barbados',
+	'bahrain': 'Bahrain',
+	'belize': 'Belize',
```
[the remark this answers is not in the document]
Comment (Iva-khar) [reply location_register/management/commands/constants.py:11]: done

File: data_ocean/converter.py (hunk 1 above)
Comment (Iva-khar) [reply data_ocean/converter.py]: done

**Review by Andrii17s — APPROVED** [review]

File: location_register/converter/address.py [hunk 4]
```diff
@@ -9,7 +9,7 @@ def __init__(self):
         super().__init__()
 
     def save_or_get_country(self, name):
```
Comment (roman-tiukh) [inline location_register/converter/address.py:11]: in this function?
is this duplicate of Converter.save_or_get_country?

**Review by roman-tiukh — CHANGES_REQUESTED** [review]
Check this:
- DeclarationConverter.find_country
- load_companies_sanctions.py
- AddressConverter.save_or_get_country
- load_person_sanctions.py
- add_country_ukrainian_name.py
- and maybe in some other places

File: location_register/converter/address.py (hunk 4 above)
Comment (Iva-khar) [reply location_register/converter/address.py:11]: yes, it's just duplicate.
`address.py` - deleted;
usages - `ukr_company.py, ukr_company_full.py, pep.py` - modificated

**Comment by Iva-khar** [comment]
> Check this:
> 
> * DeclarationConverter.find_country
> * load_companies_sanctions.py
> * AddressConverter.save_or_get_country
> * load_person_sanctions.py
> * add_country_ukrainian_name.py
> * and maybe in some other places
Checked. Besides checked all usages of `Country` model. Additionaly modificated `load_companies_sanctions.py` & `load_person_sanctions.py`

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**